src/style.css

```
@@ -1,3 +1,78 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav_item {
+  margin-right: 20px;
+  padding: 0;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-decoration: none;
+  display: inline-block;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+content: "";
+position: absolute;
+left: 0;
+bottom: 0;
+width: 100%;
+height: 4px;
+background-color: #00acdc;
+border-radius: 8px;
+}
+
+.nav_link[data-qa="hover"]:hover {
```

```suggestion
.nav_link:hover {
```

Don't use data-qa attributes for styling, they are needed to test the code